### PR TITLE
browser-based terminal

### PR DESCRIPTION
**Pull request by TownLake**

### CODE REVIEW

File: products/cloudflare-one/src/content/tutorials/ssh.md [hunk 1]
```diff
@@ -156,3 +158,13 @@ Host azure.widgetcorp.tech
 ```
 
 You can now test the SSH flow by running a command to reach the service. When the command is run, `cloudflared` will launch a browser window to prompt you to authenticate with your identity provider before establishing the connection from your terminal.
+
+### Browser-rendered terminal
+
+Cloudflare can render certain a terminal for your SSS session in your browser without the need for client software or end user configuration changes.
```
Comment (kkoenig) [inline products/cloudflare-one/src/content/tutorials/ssh.md]: SSH* session

Comment (kkoenig) [inline products/cloudflare-one/src/content/tutorials/ssh.md]: "render certain" looks like typo

File: products/cloudflare-one/src/content/tutorials/ssh.md [hunk 2]
```diff
@@ -156,3 +158,13 @@ Host azure.widgetcorp.tech
...
+
+To enable, navigate to the application page of the Access section in the Cloudflare for Teams dashboard. Click **Edit** and select the Settings tab. In the `cloudflared settings` card, toggle `Enable browser rendering` to on.
+
+![Auto Auth](../static/documentation/applications/render-browser.png)
+
+Once enabled, when users authenticate and visit the URL of the application, Cloudflare will render a terminal in their browser.
```
Comment (kkoenig) [inline products/cloudflare-one/src/content/tutorials/ssh.md:170]: Is it worth including a recommendation that users set up a unique hostname unique access application?

Or we could include in troubleshooting section

File: products/cloudflare-one/src/content/tutorials/ssh-browser.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+updated: 2021-04-15
+category: 🔐 Zero Trust
+---
+
+# Render a terminal in the browser for SSH connections
+
+Cloudflare can render certain a terminal for your SSS session in your browser without the need for client software or end user configuration changes.
```
Comment (kkoenig) [inline products/cloudflare-one/src/content/tutorials/ssh-browser.md]: SSS -> SSH?

**Review by abracchi-tw — APPROVED** [review]

**Pull request merged**